src/style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  margin: 0;
+}
+
+.box {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100px;
+  width: 100px;
+  margin: auto;
+}
+
+.stars {
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  height: 16px;
+  justify-content: space-between;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-size: cover;
+  width: 13px;
+  height: 12px;
```

Comment:
`stars__star` element has different dimensions in figma:
![image](https://user-images.githubusercontent.com/60885495/125261528-0485fc80-e30a-11eb-89a4-2209de6b548d.png)

You need to find a way to control background image size and position within this box.